Symfony/CS/Tokenizer/TokensAnalyzer.php

```diff
@@ -211,7 +211,84 @@ public function isArrayMultiLine($index)
     }
 
     /**
-     * Check if there is a lambda function under given index.
+     * Returns the attributes of the method under the given index.
+     *
+     * @param int $index Token index of the method (T_FUNCTION)
+     *
+     * @return array {
+     *     @var int|null $visibility T_PRIVATE, T_PROTECTED or T_PUBLIC
```

Comment:
The funny thing is, you're the last editor of this part: https://github.com/phpDocumentor/fig-standards/blame/master/proposed/phpdoc.md#L709-L710

But yeah, I know, it was a simple renaming commit ;)